### PR TITLE
Feature : Add open pr on github from pr description node

### LINKED ISSUE #6000: Allow to open PR on web from PR view when showing multiple
Here I really miss a way to click on the 🌐 icon to open the PR externally:

image

### PR DESCRIPTION
**Pull request by Santhoshmani1**
Closes  #6000 

- Pull requests can now opened on github from pr description nodes

## Preview
Screenshot 2024-05-27 210617

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2180,6 +2180,11 @@
 					"group": "inline@2",
 					"when": "view =~ /(pr|prStatus):github/ && viewItem =~ /description/"
 				},
+				{
+					"command": "pr.openPullRequestOnGitHub",
+					"group": "inline@3",
+					"when": "view =~ /(pr|prStatus):github/  && viewItem =~ /description/"
```
Comment (alexr00) [inline package.json:2186]: Could we also limit this action to show only when there's more than one active PR? The original issue is that when there's more than one active PR then there's no easy way to open github.com.

Comment (alexr00) [reply package.json:2186]: See here for an example of how to check the count of active PRs:

https://github.com/Santhoshmani1/vscode-pull-request-github/blob/1d12381dd4266025b54ad2942ccea1d4bf890e9b/package.json#L718-L719

**Review by alexr00 — CHANGES_REQUESTED** [review]
Thanks for the PR! You are correct that this was a good place to start. I have one comment:

File: package.json (hunk 1 above)
Comment (Santhoshmani1) [reply package.json]: I have modified the openPrOnGithub Icon visibility to show only when multiple active pull requests are present.

File: package.json [hunk 2]
```diff
@@ -2180,6 +2180,11 @@
 					"group": "inline@2",
 					"when": "view =~ /(pr|prStatus):github/ && viewItem =~ /description/"
 				},
+				{
+					"command": "pr.openPullRequestOnGitHub",
+					"group": "inline@3",
+					"when": "view =~ /(pr|prStatus):github/  && viewItem =~ /description/ && github:activePRCount >= 2"
```
Comment (alexr00) [inline package.json:2186]: One more thing I didn't notice before: I think this should only show in the prStatus view:

```suggestion
					"when": "view == prStatus:github && viewItem =~ /description/ && github:activePRCount >= 2"
```

**Review by alexr00 — CHANGES_REQUESTED** [review]

**Review by alexr00 — APPROVED** [review]
Looks good, thank you! I will merge this after the 0.92.0 release.

**Review by Tyriar — APPROVED** [review]

**Pull request merged**